### DISCUSSION TITLE
Is it possible to use dark mode with bold colours rather than pastel?

### DISCUSSION THREAD

**Discussion by Ruberine** [jgraph/drawio-desktop#1391] [Q&A]
I used to be able to use dark mode and still have bold colours, why have they all become pastel colours? Here's my colour picker now, I don't know why I can't get bold colours.
![image](https://github.com/jgraph/drawio-desktop/assets/68415990/d3891b23-0355-4b6c-ba9b-954c9fedf850)

**Comment by davidjgraph** [comment]
We're going to revert this for desktop 21.6.5. The new colour system is really for cases where there's a lot of sharing between users, obviously that doesn't happen on desktop.

If you want the new, online CSS filter, use this in the desktop config:

```
{
  "enableCssDarkMode": true
}
```
